src/seedu/addressbook/parser/Parser.java

```diff
@@ -1,28 +1,14 @@
 package seedu.addressbook.parser;
 
-import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.addressbook.common.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+import seedu.addressbook.commands.*;
```

Comment:
The coding standard for CS2103 specifies to import just the specific classes you need. If this happens automatically, you can change "code style -> Java -> imports" and change the setting